docs/release-notes/8.4.5-5.md

```diff
@@ -0,0 +1,116 @@
+# Percona XtraDB Cluster 8.4.5-5 (2025-07-14)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in both cloud and on-premises environments, including public, private, and hybrid setups. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Percona XtraDB Cluster 8.4.5-5 is based on Percona Server for MySQL and includes all improvements and bug fixes available in the [Percona Server for MySQL 8.4.5-5 (2025-05-29) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.5-5.html).
+
+### Percona XtraDB Cluster 8.4.5-5
+
+Improves State Snapshot Transfer (SST) failure diagnostics. `garbd` now uses distinct exit codes to differentiate between donor exit, SST script failure, and garbd-initiated termination, making SST issues easier to identify and debug.
+
+### Percona Server for MySQL 8.4.5-5
+
+* Updates the C++ level of the KMPI library to enhance error handling capabilities.
+
+* Improves optimizer behavior by restoring correct handling of const tables in `test_quick_select()`. A MySQL Upstream refactor (commit [9a13c1c](https://github.com/percona/percona-server/commit/9a13c1c6971f4bd56d143179ecfb34cca8ecc018)) removed the `QEP_TAB` dependency, causing `get_quick_record_count()` to no longer pass const table information. This could lead to suboptimal range scan boundaries. The applied patch resolves the issue by explicitly passing `const_tables` to `test_quick_select()`, ensuring consistent behavior with the pre-refactor logic.
+
+The latest MyRocks storage engine incorporates code based on RocksDB version 9.3.1. Percona has applied minor modifications to the original RocksDB codebase. Check the list of modifications at [https://github.com/percona/rocksdb/](https://github.com/percona/rocksdb/commit/2864cd95ca72bc4a2e93fe461dbd980f8e2357be).
```

Comment:
I know that this part comes from PS, but in PXC there is no MyRocks storage engine